components/blocks/Features.tsx

```diff
@@ -31,25 +37,39 @@ export function FeatureBlock({ data, index }) {
           >
             {data.text}
           </p>
-          {data.actions && <Actions items={data.actions} />}
+          <div className="flex lg:flex-row">
+            {data.actions && <Actions items={data.actions} />}
+          </div>
         </div>
         {data.media && data.media[0] && (
           <div
-            className={`w-full min-w-0 lg:w-1/2 ${(data.media[0].image || data.media[0].src) &&
-              'rounded-lg shadow-panel overflow-hidden bg-gradient-to-br from-blue-800 via-blue-900 to-slate-900'
-              }`}
+            className={`relative min-w-0 lg:w-1/2 ${(data.media[0].image || data.media[0].src) && ''}`}
           >
             {data.media && data.media[0].image && (
-              <img
-                src={data.media[0].image}
-                alt={data.headline}
-                className="w-full h-auto"
-              // width="1120px"
-              // height="800px"
-              />
+              <>
+                <img
+                  src={data.media[0].image}
+                  alt={data.headline}
+                  className="w-full h-auto rounded-lg shadow-panel overflow-hidden bg-transparent"
+                />
+              </>
```

Comment:
any reason we aren't using [Next image](https://nextjs.org/docs/pages/api-reference/components/image)?

Maybe make a PBI to use it later

Reply:
https://github.com/tinacms/tina.io/issues/1832 

✅